components/FormConnect/FormConnect.tsx

```diff
@@ -46,6 +47,13 @@ const FormConnect = () => {
       userForm.email !== "" &&
       userForm.password !== ""
     ) {
+      if (!validateEmail(userForm.email)) {
+        enqueueSnackbar("Veuillez entrer une adresse mail valide", {
+          variant: "warning",
+          autoHideDuration: 3000,
```

Comment:
Why can't you configure in SnackbarProvider and overwrite if you need it. See this doc : https://notistack.com/api-reference
<img width="545" alt="image" src="https://github.com/theo-code33/short-url-app/assets/90395803/60a12fc8-4e4b-4cd2-ad1b-45bedf9237af">